analysis/wwz/wwz4l.py

```diff
@@ -691,8 +698,8 @@ def process(self, events):
             mll_01 = (l0+l1).mass
             mllll = (l0+l1+l2+l3).mass
             scalarptsum_lep = l0.pt + l1.pt + l2.pt + l3.pt
-            scalarptsum_lepmet = l0.pt + l1.pt + l2.pt + l3.pt + met.pt
-            scalarptsum_lepmetjet = l0.pt + l1.pt + l2.pt + l3.pt + met.pt + ak.sum(goodJets.pt,axis=-1)
+            scalarptsum_lepmet = l0.pt + l1.pt + l2.pt + l3.pt + met.pt_new
```

Comment:
I have to say, this feels a bit dangerous. If someone new picks up the code, they might use met.pt and met.phi without realizing they should be using the "new". 

I assume it's becuase there is not a way to replace the original,  and I cannot think of a good way to work around this danger right now. So let's just keep it in mind and try to keep thinking of solutions.

Reply:
Eventually, I think just making a new met object would be better. I didn't like this solution either, but it works in the meantime